lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -2716,7 +2716,7 @@ public function getCurrentTimestampSQL()
      *
      * @param int $level
      *
-     * @return string
+     * @return string|int
```

Comment:
Should it be fixed or silenced instead? Doesn't look like it's by design (the SQLite platform impelementation).

Reply:
I can change those ints to strings, but I really have no idea of consequences (either in DBAL or for consumers). SqlAnywherePlatoform returns ints too... Maybe better leave this to 3.x?

Reply:
I'm all for leaving it for 3.x where we could fix the actual code instead of complicating the API documentation.

Reply:
Please ignore instead.